projects/storefrontlib/src/shared/components/split-view/index.ts

```diff
@@ -3,3 +3,4 @@ export * from './split-view.module';
 export * from './split-view.service';
 export * from './split/index';
 export * from './view/index';
+export * from './testing/spit-view-testing.module';
```

Comment:
`split-view-testing.module` is private testing util not to be exposed in public API yet (it's incomplete solution). Please import from absolute path in tests, if needed.